vignettes/wip/dm-db.Rmd

```diff
@@ -146,10 +187,18 @@ my_dm_total$total_loans %>%
 
 ## Persisting results
 
-- Relevant phrase, from `?dbplyr::compute`: "Force computation of a query", needs to be adapted to our needs (=multiple queries)
-- Requires write access to the database (at least for temporary tables), which we don't have on relational.fit
-    - Workaround for demonstration only: copy to local SQLite database, see below for details on `copy_dm_to()`
+After adding columns to link tables or calculating summaries of data, we might want these changes to the database to persist. 
+#FIXME - if compute stores results in temporary tables, how are they made permanent or inserted into the database itself?
+
+To force a dm to execute the SQL query generated by the operations it has performed, we use the `compute()` function, originally implemented in {dbplyr}, but reimplimented in {dm} to provide the same functionality.
```

Comment:
The generic is provided by {dbplyr}, the {dm} package implements methods for dm and zoomed_dm objects. Here we can perhaps simplify:

```suggestion
To force a dm to execute the SQL query generated by the operations it has performed, we call the `dbplyr::compute()` method on the dm object.
```